x-pack/legacy/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/create_analytics_form/create_analytics_form.tsx

```diff
@@ -118,7 +118,8 @@ export const CreateAnalyticsForm: FC<CreateAnalyticsFormProps> = ({ actions, sta
       delete jobConfig.model_memory_limit;
       const resp = await ml.dataFrameAnalytics.estimateDataFrameAnalyticsMemoryUsage(jobConfig);
       setFormState({
-        modelMemoryLimit: resp.expected_memory_without_disk,
+        modelMemoryLimit:
+          resp.memory_estimation && resp.memory_estimation.expected_memory_without_disk,
```

Comment:
nit, this could be `resp.memory_estimation?.expected_memory_without_disk`

Reply:
Updated in f8ec99d19df0f1b1e985b50cf2f4c943516c8a5e